exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcSpanExporter.java

```diff
@@ -291,7 +291,7 @@ protected Builder fromConfigMap(
       String metadataValue = getStringProperty(KEY_METADATA, configMap);
       if (metadataValue != null) {
         for (String keyValueString : Splitter.on(';').split(metadataValue)) {
-          final List<String> keyValue = Splitter.on('=').splitToList(keyValueString);
+          final List<String> keyValue = Splitter.on('=').limit(2).splitToList(keyValueString);
```

Comment:
Should we also add `trimResults` `omitEmptyStrings`? These are HTTP headers so at least keys cannot have spaces or be empty. I'm less sure about trimming the values or the key only, but I think usually a trailing space on a value will be a bug not a feature.

Reply:
added